<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>klMainWindow</class>
 <widget class="QMainWindow" name="klMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SRP v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/img/logo2.png</normaloff>:/img/img/logo2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QGroupBox{
font: 10pt &quot;Calibri&quot;;
color: #145c7f;
border-radius:12px;
background-color: #64b2d5;
}

QMainWindow{
background-color:#145c7f;
font: 10pt &quot;Calibri&quot;;
}

QLabel{
font: 10pt &quot;Calibri&quot;;
color: #64b2d5;
}

QLineEdit{
font: 75 10pt &quot;Calibri&quot;;
color: #145c7f;
background-color: rgb(246, 246, 246);
border-color: #64b2d5;
border-radius:6px;
}

QPushButton{
font: 75 10pt &quot;Calibri&quot;;
font-weight: bold;
color: #145c7f;
background-color: rgb(246, 246, 246);
border-style: solid;
border-radius:12px;
width: 120px;
height:24px;
}


QPushButton:hover{
color: #145c7f;
background-color: rgb(246, 246, 246);
border-color: #145c7f;
border-width: 2px;
}

QPushButton:pressed{
color:  rgb(246, 246, 246);
border-color:   rgb(246, 246, 246);
background-color: #145c7f;
}

QTextEdit{
font: 75 10pt &quot;Calibri&quot;;
color: #145c7f;
background-color: rgb(246, 246, 246);
border-color: #64b2d5;
border-radius:6px;
}

QTableWidget{
font: 75 10pt &quot;Calibri&quot;;
color: #145c7f;
background-color: rgb(246, 246, 246);
border-color: #64b2d5;
border-radius:12px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_22">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/img/logo_mini.jpg);
border-color:rgb(160, 160, 160);
border-style: solid;</string>
             </property>
             <property name="title">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Calibri&quot;;
color: rgb(57, 57, 57);</string>
               </property>
               <property name="text">
                <string>Numer przesyłki: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leNumer">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>178</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pbKonto">
          <property name="text">
           <string>Moje konto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbWyslij">
          <property name="text">
           <string>Wyślij paczke</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="minimumSize">
         <size>
          <width>321</width>
          <height>266</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>266</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QTableWidget" name="tbTabelka">
           <property name="minimumSize">
            <size>
             <width>301</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Arial&quot;;
selection-color: rgb(0, 0, 0);
selection-background-color: #20dad8;</string>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>nr. paczki</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>nadawca / odbiorca</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="minimumSize">
         <size>
          <width>192</width>
          <height>266</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>192</width>
          <height>266</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
background-image: url(:/img/img/statusbar1.jpg);
}


@RadioButton{
font: 10pt &quot;Calibri&quot;;
font-color:#145c7f;
}

QLineEdit{
font: 75 10pt &quot;Calibri&quot;;
color: rgb(246, 246, 246);
background-color: #64b2d5;
height: 15px;
width:140px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_21">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #145c7f;</string>
             </property>
             <property name="text">
              <string>Historia przesyłki</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb0_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(57, 57, 57)</string>
             </property>
             <property name="text">
              <string>gotowa do odebrania</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="ledata0_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rb1_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(57, 57, 57)</string>
             </property>
             <property name="text">
              <string>odebrana od nadawcy</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="ledata1_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rb2_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(57, 57, 57)</string>
             </property>
             <property name="text">
              <string>przyjęta w oddziale</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <spacer name="horizontalSpacer_15">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="ledata2_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rb3_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(57, 57, 57);</string>
             </property>
             <property name="text">
              <string>przekazana do doręczenia</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <spacer name="horizontalSpacer_16">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="ledata3_2">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rb4_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(57, 57, 57)</string>
             </property>
             <property name="text">
              <string>dostarczona</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <spacer name="horizontalSpacer_17">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="ledata4_2">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
